kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CommandLine.scala

```diff
@@ -137,4 +194,94 @@ object CommandLine {
           .text("The engine share level this engine belong to."))
   }
 
+  private def batchCmd(builder: OParserBuilder[CliConfig]): OParser[_, CliConfig] = {
+    import builder._
+    cmd("batch").action((_, c) => c.copy(resource = ControlObject.BATCH))
+  }
+
+  private def getBatchCmd(builder: OParserBuilder[CliConfig]): OParser[_, CliConfig] = {
+    import builder._
+    cmd("batch").action((_, c) => c.copy(resource = ControlObject.BATCH))
+      .children(
+        arg[String]("<batchId>")
```

Comment:
`<batchId>` or `batchId`?

Reply:
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/21362040/173237339-dacc912e-2d80-402b-8a28-3ab86a70fe7f.png">

Reply:
got it. thanks